torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -470,6 +402,25 @@ def batch_norm_inference(input, weight, bias, mean, var, momentum, eps):
                 output = batch_norm_post_process(output, shape, weight, bias)
                 return g.call_function(target_wrappers.pack_to_tuple, (output,))
 
+            if node.target in TTNN_POINTWISE_UNARY_OPS:
+                code = TTNN_POINTWISE_UNARY_OPS[node.target]
+
+            def unsqueeze_to_2d(code, args=args, kwargs=kwargs):
+                ndims = len(node.meta["val"].size())
+                result = g.call_function(code, args, kwargs)
+
+                if ndims == 1:
+                    result = g.call_function(ttnn.to_layout, (result, TtnnRowMajorLayout()))
+                    result = g.call_function(ttnn.squeeze, (result, 0))
+
+                return result
+
+            if node.target in TTNN_POINTWISE_UNARY_OPS:
+                return unsqueeze_to_2d(TTNN_POINTWISE_UNARY_OPS[node.target])
+
+            if node.target == torch.ops.aten.round.default:
+                return unsqueeze_to_2d(ttnn.round, (args[0],), {"decimals": 0})
```

Comment:
Must update ttnn op binding to default the argument to 0?

Reply:
That would be great!  I once tried at tenstorrent/tt-metal#13851 but in vain.